src/components/ProductListItem/index.tsx

```diff
@@ -26,7 +26,7 @@ class ProductListItem extends BaseComponent<ProductListItemProps> {
           class="w-full h-full m-0 border-0"
           src={this.image.src}
           resolutions={this.image.resolutions}
-          lazy={true}
```

Comment:
Das ist, um den Test zu fixen? Ich habe dazu in meinem PR einen anderen Fix, mit dem man nicht auf "lazy" verzichten braucht: https://github.com/e-Spirit/fsxa-ui/pull/54/commits/801590b073693d5ae25259ffeb25cb9ace702a07